### DISCUSSION TITLE
How are aliases meant to be used?

### DISCUSSION THREAD

**Discussion by grimnight** [pimalaya/himalaya#519] [Q&A] [answered]
I just installed Himalaya to try it out. I have been trying to use folder aliases, but I can't get them to work. Am I doing something wrong or have I misunderstood folder aliases?

Excerpt from my config (sanitized):
```toml
[accounts.account_name]
default = true
email = "grimnight@email"

display-name = "GrimNight"
backend.type = "maildir"
backend.root-dir = "~/.local/share/mail/grimnight/"
backend.maildirpp = false
message.send.backend.type = "sendmail"
message.send.backend.cmd = "/usr/sbin/sendmail"

folder.alias.inbox = "INBOX"
folder.alias.old = "Archives"
```

And my attempts:
```shell
~
❯ himalaya envelope list -f old
Error:
   0: cannot find maildir matching name old

Note: Run with --debug to enable logs with spantrace.
Note: Run with --trace to enable verbose logs with backtrace.

~
❯ himalaya envelope list -f inbox
Error:
   0: cannot find maildir matching name inbox

Note: Run with --debug to enable logs with spantrace.
Note: Run with --trace to enable verbose logs with backtrace.
```

The folder aliases don't seem to work, unless I have misunderstood them.

Both `himalaya envelope list` and `himalaya envelope list -f INBOX` work fine.

**Comment by soywod** [comment]
It is definitely supposed to be used the way you discribed it, so I'm afraid there is a regression on the Maildir backend. Let me transfer this discussion to an issue.

**Reply by soywod** [reply, marked as answer]
Refs: #526

**Reply by soywod** [reply]
It has been fixed, see https://github.com/pimalaya/himalaya/issues/526#issuecomment-2553041780.
